e2e/src/usecases/ChooseYourAdventure.js

```diff
@@ -9,7 +9,7 @@ export default ChooseYourAdventure = () => {
       newInstance: true,
       launchArgs: {
         onboardingOverrides:
-          'EnableBiometry,ProtectWallet,PhoneVerification,CloudBackupSetup,CloudBackupRestore',
+          'EnableBiometry,ProtectWallet,PhoneVerification,CloudBackupSetupInOnboarding,CloudBackupRestore',
```

Comment:
`CloudBackupSetupInOnboarding` shouldn't be in here right if we want to keep e2e tests as is?